nym-vpn/ui/src-tauri/src/fs/data.rs

```diff
@@ -22,3 +23,12 @@ pub struct AppData {
     pub entry_node: Option<NodeConfig>,
     pub exit_node: Option<NodeConfig>,
 }
+
+#[derive(Default, Serialize, Deserialize, Debug, Clone, TS)]
+#[ts(export)]
+pub struct Country {
+    pub name: &'static str,
+    pub code: &'static str,
```

Comment:
It might be better to use directly owned `String` here, as in the future we will get the flag list data from external source
So we cannot use static literal string type (`&'static str`) from that point